source/docs/software/wpilib-overview/3rd-party-libraries.rst

```diff
@@ -14,7 +14,7 @@ Libraries
 
 `Copperforge LibCu Software Library <https://copperforge.cc/docs/software/libcu/>`__ - Library for all Copperforge devices including the Lasershark
 
-`CTRE Phoenix Toolsuite <https://www.ctr-electronics.com/control-system/hro.html#product_tabs_technical_resources>`__ - Contains TalonSRX/Victor SPX Libraries and Phoenix Tuner program for configuring CTRE CAN devices
+`CTRE Phoenix Toolsuite <https://www.ctr-electronics.com/control-system/hro.html#product_tabs_technical_resources>`__ - Contains Talon SRX/Victor SPX/Talon FX/Pigeon Libraries and Phoenix Tuner program for configuring CTRE CAN devices
```

Comment:
At this point, just use commas. 

"Contains the TalonSRX, Victor SPX, and Talon FX libraries, alongside the Phoenix Tuner program"